MetaMask/metamask-extension
# feat: add the ConsenSys zkEVM (Linea) as a default network

## Explanation

The goal of the PR is to add the ConsenSys zkEVM network (Linea) as a default test network like Goerli and Sepolia.
I've updated all translation files.
This change can only be tested when this PR (https://github.com/MetaMask/design-tokens/pull/408) is merged and the new design tokens library version is updated on metamask-extension repo.



## Screenshots/Screencaps



### Before
<img width="309" alt="Screenshot 2023-02-23 at 11 56 19 AM" src="https://user-images.githubusercontent.com/85494462/220887399-ce698c93-e6f8-422a-bc59-2506d781cf61.png">



### After
![Screenshot 2023-03-14 at 6 01 16 PM](https://user-images.githubusercontent.com/85494462/225082245-d0a883f6-3784-43ea-9b87-9dfb0b2735df.png)




## Manual Testing Steps

Step 1:
Go to the metamask home screen and select Goerli test network in the dropdown.
![Screenshot 2023-03-14 at 6 01 16 PM](https://user-images.githubusercontent.com/85494462/225082245-d0a883f6-3784-43ea-9b87-9dfb0b2735df.png)

Step 2:
Go to this website: https://bridge.goerli.zkevm.consensys.net/send?token=ETH and send some ETH (minimum 0.1) from Goerli to Consensys zkEVM .
<img width="1908" alt="Screenshot 2023-02-23 at 12 04 31 PM" src="https://user-images.githubusercontent.com/85494462/220888987-def69536-19de-433f-bf11-6170c25eab1e.png">

Wait for transaction to be finished.

Step 3:
Go to the metamask home screen and select then new ConsenSys zkEVM test network in the dropdown.
![Screenshot 2023-03-14 at 6 01 16 PM](https://user-images.githubusercontent.com/85494462/225082245-d0a883f6-3784-43ea-9b87-9dfb0b2735df.png)

Step 4:
Go to this website: https://bridge.goerli.zkevm.consensys.net/send?token=ETH and send some ETH (minimum 0.1) from  Consensys zkEVM to Goerli.
<img width="1914" alt="Screenshot 2023-02-23 at 12 07 30 PM" src="https://user-images.githubusercontent.com/85494462/220889588-0561bce0-3e23-43fa-acea-3396084d989c.png">

Wait for transaction to be finished.

Final step:
Check if all transactions succeed on Metamask.



## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
